tst/EnergyPlus/unit/MixedAir.unit.cc

```diff
@@ -0,0 +1,150 @@
+// EnergyPlus::MixedAir Unit Tests
+
+// Google Test Headers
+#include <gtest/gtest.h>
+
+// EnergyPlus Headers
+#include <MixedAir.hh>
+#include <OutAirNodeManager.hh>
+#include <InputProcessor.hh>
+#include <DataHeatBalance.hh>
+#include <DataIPShortCuts.hh>
+#include <DataGlobals.hh>
+#include <DataStringGlobals.hh>
+#include <ObjexxFCL/gio.hh>
+#include <DataLoopNode.hh>
+
+using namespace EnergyPlus;
+using namespace EnergyPlus::MixedAir;
+using namespace EnergyPlus::OutAirNodeManager;
+using namespace EnergyPlus::InputProcessor;
+using namespace EnergyPlus::DataIPShortCuts;
+using namespace EnergyPlus::DataGlobals;
+using namespace EnergyPlus::DataLoopNode;
+using namespace ObjexxFCL;
+
+TEST( ProcessOAControllerTest, Test1 )
+{
+// Test input processing of portion of Controller:OutdoorAir object
+	//  Controller:OutdoorAir,
+	//    OA Controller 1,         !- Name
+	//    Relief Air Outlet Node 1,!- Relief Air Outlet Node Name
+	//    VAV Sys 1 Inlet Node,    !- Return Air Node Name
+	//    Mixed Air Node 1,        !- Mixed Air Node Name
+	//    Outside Air Inlet Node 1,!- Actuator Node Name
+	//    autosize,                !- Minimum Outdoor Air Flow Rate {m3/s}
+	//    autosize,                !- Maximum Outdoor Air Flow Rate {m3/s}
+
+	static bool ErrorsFound( false ); // If errors detected in input
+	static int ControllerNum( 0 ); // Controller number
```

Comment:
@jasondegraw Good question.  Probably because this is static here (which is what I copied to make this one):

https://github.com/NREL/EnergyPlus/blob/develop/tst/EnergyPlus/unit/HeatBalanceManager.unit.cc#L36

So, why are those static there?  Because it's static here and I probably copy/pasted the declaration from here:

https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/HeatBalanceManager.cc#L327

And it's not clear to me that even the one in HeatBalanceManager needs to be static because GetHeatBalanceInput is only called once (fairly certain about that, but not completely).